docs/docusaurus.config.js

```diff
@@ -1,88 +1,156 @@
-const remarkSources = require('remark-sources');
+const path = require('path');
+const {
+  rehypePlugins: themeRehypePlugins,
+  remarkPlugins: themeRemarkPlugins,
+} = require('@rasahq/docusaurus-theme-tabula');
 const { remarkProgramOutput } = require("./plugins/program_output");
 
-let versions = [];
-try {
-  versions = require('./versions.json');
-} catch (ex) {
-  // Nothing to do here, in dev mode, only
-  // one version of the doc is available
-}
-let legacyVersions = [];
+const isDev = process.env.NODE_ENV === 'development';
+
+let existingVersions = [];
+try { existingVersions = require('./versions.json'); } catch (e) { console.info('no versions.json file found') }
+
+const BASE_URL = '/docs/action-server/';
+const SITE_URL = 'https://rasa.com';
+// NOTE: this allows switching between local dev instances of rasa/rasa-x
+const SWAP_URL = isDev ? 'http://localhost:3001' : SITE_URL;
```

Comment:
why don't we make the default livedocs ports go in order of the top nav in the package.json, so that they're consistent across repos (each other docs page needs a swap url, they can't both be on one port)
rasa open source: 3000
rasa x: 3001
action server: 3002

Reply:
ah yeah, my mistake, updating it!